### PR TITLE
Add pvrtc-compressor 20150108

### PR DESCRIPTION
**Pull request by intelligide**
Specify library name and version:  **pvrtc-compressor/cci.20150108**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/pvrtc-compressor/all/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,42 @@
+cmake_minimum_required(VERSION 3.1)
```
Comment (uilianries) [inline recipes/pvrtc-compressor/all/CMakeLists.txt:1]:
```suggestion
cmake_minimum_required(VERSION 3.4)
```

CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS requires 3.4

**Comment by uilianries** [comment]
Bitbucket is under maintenance right now: https://bitbucket.status.atlassian.com/incidents/hmgnc3x4lfwh

I'll review this tomorrow.

**Review by madebr — CHANGES_REQUESTED** [review]
None of the bitbucket links are working.
What's going on over there?

**Comment by intelligide** [comment]
Bitbucket removed Mercurial Support.

**Comment by intelligide** [comment]
I was able to retrieve the sources from my conan cache. I made a backup of it: https://github.com/FrozenStormInteractive/pvrtc-compressor

**Comment by madebr** [comment]
> 
> 
> I was able to retrieve the sources from my conan cache. I made a backup of it: https://github.com/FrozenStormInteractive/pvrtc-compressor

That's nice for preservation purposes.

@jthlim
Is there a new way to fetch pvrtc-compressor not bitbucket has retired mercurial support?

**Comment by jgsogo** [comment]
Wow, the repo is gone (bitbucket remove mercurial repos time ago... ) We need an alternate URL to get the sources. cc/ @danimtb

**Comment by Croydon** [comment]
Hi @jthlim! If I'm not mistaken you are the author of pvrtc-compressor. The previous pvrtc-compressor mercurial repository got deleted on BitBucket. Could you please consider creating a new git repository on GitHub/GitLab/Bitbucket or so? 

It would be good to have a new official origin for the source code

**Comment by SSE4** [comment]
sources are still not found... is there a mirror?

**Comment by jgsogo** [comment]
We can add sources here (https://github.com/conan-io/cci-sources-backup) and retrieve them from our repo...

**Comment by SSE4** [comment]
> We can add sources here (https://github.com/conan-io/cci-sources-backup) and retrieve them from our repo...

yes, but there to retrieve them now? the URL doesn't work:
https://bitbucket.org/jthlim/pvrtccompressor/get/cf7177748ee0dcdccfe89716dc11a47d2dc81af5.tar.gz
```
The requested repository either does not exist or you do not have access. If you believe this repository exists and you have access, make sure you're authenticated.
```

**Comment by jgsogo** [comment]
@intelligide did a backup some time ago (https://github.com/FrozenStormInteractive/pvrtc-compressor), and it can also be found in different places like https://github.com/godotengine/godot/tree/master/thirdparty/pvrtccompressor or https://github.com/codywatts/PVRTCCompressor.

**Comment by jcar87** [comment]
Closing this PR as the author of the library never reached out, we don't have a canonical location for the source code. Assuming that the library is also unmaintained as well. Please open an issue  to track this if needed!

**Pull request closed without merging**